storageClient/files.go

```diff
@@ -0,0 +1,49 @@
+package storageClient
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+func (c *StorageClient) UploadFile(url string, encryptedFileName string, checksum string, size int) (int, error) {
+	file, err := os.Open(encryptedFileName)
+	if err != nil {
+		return 0, err
+	}
+	defer file.Close()
+	req, err := http.NewRequest("PUT", url, file)
+	if err != nil {
+		return 0, err
+	}
+	req.Header.Set("Content-MD5", checksum)
+	req.Header.Set("Content-Type", "application/octet-stream")
+	req.ContentLength = int64(size)
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil || resp.StatusCode < 200 || resp.StatusCode >= 300 {
+		return resp.StatusCode, err
+	}
+	return 0, nil
+}
+
+func (c *StorageClient) DownloadFile(url string, encryptedFileName string) (string, error) {
```

Comment:
comment, and please extract these parameters into a struct

Reply:
please make these functions in a package called file, not methods for StorageClient.